src/frontend/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub mod expr;
 pub mod handler;
 pub use handler::PgResponseStream;
 mod observer;
-mod optimizer;
+pub mod optimizer;
```

Comment:
👀

Reply:
Seems we need to export this module to make it doc-testable. 👀